### PR TITLE
nih validation tests

### PR DESCRIPTION
**Pull request by ProOlha**

### CODE REVIEW

File: tests/nihValidation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+        await nih.userInfoPage.clickContactInfo()
+        await nih.userInfoPage.checkValidation(nihLocators.officialDocumentValidationLocator)
+        
+    })
+
+    test('Payment displayed Validation', async({page}) => {
+        const nih = new NihApp(page)
+        await nih.userInfoPage.selectTestResultField('True')
+        await nih.userInfoPage.clickContactInfo()
+        await page.locator(nihLocators.cardDisplayedLocator).waitFor()
```
Comment (IvanKSPsoft) [inline tests/nihValidation.spec.ts:218]: Avoid using page
Use specific app and specific methods

**Review by IvanKSPsoft — CHANGES_REQUESTED** [review]
Avoid using page
Use specific app and specific methods

File: tests/nihValidation.spec.ts (hunk 1 above)
Comment (ProOlha) [reply tests/nihValidation.spec.ts]: Method have added and tested

File: tests/nihValidation.spec.ts [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    test('DOB Validation', async({page}) => {
+        const nih = new NihApp(page)
+        await nih.userInfoPage.inputDOB('01012000')
+        await nih.userInfoPage.clickContactInfo()
+        await page.locator(nihLocators.dobLocator).fill('')
+        await nih.userInfoPage.inputFirstName('test')
+        await nih.userInfoPage.inputLastName('test')
+        await nih.userInfoPage.clickContactInfo()
+        await nih.userInfoPage.checkDobValidation(nihLocators.dobFieldValidationLocator)
+        await page.locator(nihLocators.dobFieldValidationLocator).waitFor()
```
Comment (IvanKSPsoft) [inline tests/nihValidation.spec.ts:43]: move to object

**Review by IvanKSPsoft — CHANGES_REQUESTED** [review]

**Pull request merged**